src/main/java/org/traccar/protocol/MegastekProtocolDecoder.java

```diff
@@ -275,9 +275,13 @@ private Position decodeOld(Channel channel, SocketAddress remoteAddress, String
             .or().text(" ")
             .groupEnd("?").text(",")
             .number("(d+)?,")                    // rfid
+            .groupBegin()                        // ext accessories
+            .number("([01])")                    // charging
+            .number("(d)?")                      // belt status
+            .groupEnd("?")
```

Comment:
I would replace it with something like this because it really looks like a single field:

```suggestion
            .number("([01])(d)?,").optional() // charge and belt status
```